recipe/meta.yaml

```diff
@@ -38,8 +38,8 @@ outputs:
         - libnuma
       run:
       run_constrained:
-        - cudatoolkit {{ cuda_compiler_version }}  # [cuda_compiler_version not in ("None", "11.2")]
-        - cudatoolkit >=11.2,<12                   # [cuda_compiler_version == "11.2"]
+        - cudatoolkit {{ cuda_compiler_version }}|{{ cuda_compiler_version }}.*  # [cuda_compiler_version not in ("None", "11.2")]
+        - cudatoolkit >=11.2,<12                                                 # [cuda_compiler_version == "11.2"]
```

Comment:
Addressing issue ( https://github.com/conda-forge/ucx-split-feedstock/issues/120 ) by following [this logic from `nvcc`]( https://github.com/conda-forge/nvcc-feedstock/blob/04b48c9d3923aa71a3e021ddef7d04e6e12502c9/recipe/meta.yaml#L40-L46 )